ocean_lib/ocean/test/test_ocean_compute.py

```diff
@@ -0,0 +1,537 @@
+    assert isinstance(cluster_dict, dict), "The cluster is not a dict."
+    assert (
+        "type" in cluster_dict and cluster_dict["type"] == "Kubernetes"
+    ), "Type does not belong to the attributes."
+    assert (
+        "url" in cluster_dict and cluster_dict["url"] == "http://10.0.0.17/my_cluster"
+    ), "Url does not belong to the attributes."
+
+
+def test_build_container_attributes():
+    data_provider = DataServiceProvider
+    config = Ocean.config
+    compute = OceanCompute(config=config, data_provider=data_provider)
+    container_dict = compute.build_container_attributes(
+        "node", "best_tag", "entrypoint.exe"
+    )
+    assert container_dict, "Container dictionary is None."
+    assert isinstance(container_dict, dict), "The container is not a dict."
+    assert (
+        "image" in container_dict and container_dict["image"] == "node"
+    ), "Image does not belong to the attributes."
+    assert (
+        "tag" in container_dict and container_dict["tag"] == "best_tag"
+    ), "Tag does not belong to the attributes."
+    assert (
+        "entrypoint" in container_dict
+        and container_dict["entrypoint"] == "entrypoint.exe"
+    ), "Entrypoint does not belong to the attributes."
+
+
+def test_build_server_attributes():
+    data_provider = DataServiceProvider
+    config = Ocean.config
+    compute = OceanCompute(config=config, data_provider=data_provider)
+    server_dict = compute.build_server_attributes(
+        "123", "foo_server_type", 4, 4, "20", "20", 30
+    )
+    assert server_dict, "Server dictionary is None."
+    assert isinstance(server_dict, dict), "The server is not a dict."
+    assert (
+        "serverId" in server_dict and server_dict["serverId"] == "123"
+    ), "serverId does not belong to the attributes."
+    assert (
+        "serverType" in server_dict and server_dict["serverType"] == "foo_server_type"
+    ), "serverType does not belong to the attributes."
+    assert (
+        "cpu" in server_dict and server_dict["cpu"] == 4
+    ), "CPU does not belong to the attributes."
+    assert (
+        "gpu" in server_dict and server_dict["gpu"] == 4
+    ), "GPU does not belong to the attributes."
+    assert (
+        "memory" in server_dict and server_dict["memory"] == "20"
+    ), "Memory does not belong to the attributes."
+    assert (
+        "disk" in server_dict and server_dict["disk"] == "20"
+    ), "Disk does not belong to the attributes."
+    assert (
+        "maxExecutionTime" in server_dict and server_dict["maxExecutionTime"] == 30
+    ), "Execution time does not exist."
+
+
+def test_build_service_provider_attributes():
+    data_provider = DataServiceProvider
+    config = Ocean.config
+    compute = OceanCompute(config=config, data_provider=data_provider)
+
+    cluster_dict = compute.build_cluster_attributes(
+        "Kubernetes", "http://10.0.0.17/my_cluster"
+    )
+    assert cluster_dict, "Cluster dictionary is None."
```

Comment:
can you please collect these asserts together? I would create a dictionary containing the expected values and compare the entire dictionary instead of comparing each item separately.